src/ImageProcessor.Web.Plugins.AzureBlobCache/AzureImageService.cs

```diff
@@ -59,7 +60,16 @@ public class AzureImageService : IImageService
         /// </returns>
         public async Task<byte[]> GetImage(object id)
         {
-            CloudBlockBlob blockBlob = this.blobContainer.GetBlockBlobReference(id.ToString());
+            CloudBlockBlob blockBlob = null;
```

Comment:
I don't know what this is supposed to be fixing. I can always hit my container via the config.

Reply:
Remember please that this was not built for Umbraco. If a change is required for Umbraco any fixes should be applied there.

Reply:
Okay, I got that wrong. But are you sure this little extra check is not valuable for regular usage?
If you really do not want this in your package, I'll create a new PR for just the _AzureBlobCache_ request. Btw. are you fine with that one?

Reply:
I don’t know what the cache change I supposed to achieve. We already have config for choosing whether to stream or redirect where both options are viable.